cogs/automod.py

```
@@ -0,0 +1,221 @@
+import discord
+from discord.ext import commands
+import re
```

Ruff (0.11.9)

`re` imported but unused

Remove unused import: `re`

(F401)